### ISSUE TITLE
Cannot Upload Settings First Time

### ISSUE THREAD

**Issue by BenMGilman** [shanalikhan/code-settings-sync#161]
**Visual Studio Code Version :** 1.8.0
**Code Sync Settings Version :** 2.3.8
**Operating System :** MacOS 10.12.2
**Occurs On:** Upload
**Proxy Enabled:** No

First time user. Generated and tried with 4 different GIST tokens to get it to work. Have uninstalled and reinstalled the extensions three times, following the guide step by step. I get this error every time

```
//Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/electron-browser/workbench.main.js:127 [Extension Host] {"message":"Validation Failed","errors":[{"resource":"Gist","code":"missing_field","field":"files"}],"documentation_url":"https://developer.github.com/v3/gists/#edit-a-gist"}
```

**Comment by shanalikhan** [comment]
can you please share the image of your user folder of code and snippets folder?

**Comment by BenMGilman** [comment]
It _did_ have to do with snippets. My JavaScript snippet file was empty, and all I had to do was add an empty object (`{}`) for it to work

**Comment by shanalikhan** [comment]
i will fix this in the upcoming version.

**Comment by shanalikhan** [comment]
Published to insiders - Code 1.9
Version 2.4
If all works well i will publish it to normal release

**Comment by shanalikhan** [comment]
Released for all users Version 2.4.2
https://github.com/shanalikhan/code-settings-sync/releases/tag/v2.4.2

let me know if there is any error
